### ISSUE TITLE
Misleading description?

### ISSUE THREAD

**Issue by qds-x** [ansible-collections/kubernetes.core#681]
https://github.com/ansible-collections/kubernetes.core/blob/7c4ec3b9822770d83a8717f73368f9e51f4dbd9d/plugins/modules/k8s_cluster_info.py#L22

**Comment by gravesm** [comment]
@qds-x Could you provide some more information about what you find misleading?

**Comment by qds-x** [comment]
@gravesm Yes sorry, it was misleading to me because I took it to mean that the purpose of the module is to read arbitrary objects from the API server.

If I understand correctly it actually only reads certain objects to provide information about the cluster.

For context I came to this repo to try to understand what the equivalent kubectl commands to the cluster_info module would be - it seems to go well beyond 'kubectl cluster-info' - and left none the wiser.

**Comment by yurnov** [comment]
Hi @qds-x, 

here `Use the Kubernetes Python client` means that the underlying code has `import kubernetes` to use [kubernetes python client](https://github.com/kubernetes-client/python). 

So, I don't see a misleading description, but agree that it's the description is not clear at all
